buildroot/share/PlatformIO/variants/BIGTREE_SKR_PRO_1v1/PeripheralPins.c

```diff
@@ -91,7 +91,13 @@ const PinMap PinMap_I2C_SDA[] = {
   {PB_11, I2C2, STM_PIN_DATA(STM_MODE_AF_OD, GPIO_NOPULL, GPIO_AF4_I2C2)},
   {PC_9,  I2C3, STM_PIN_DATA(STM_MODE_AF_OD, GPIO_NOPULL, GPIO_AF4_I2C3)},
   #if STM32F4X_PIN_NUM >= 144  //144 pins mcu, 114 gpio
-    {PF_0,  I2C2, STM_PIN_DATA(STM_MODE_AF_OD, GPIO_NOPULL, GPIO_AF4_I2C2)},
+    if(STM32F4X_PIN_NUM >= 176){
```

Comment:
You can't use `if` here. It is not valid C++. Look carefully! You should be using the preprocessor `#if`.

Reply:
> You can't use `if` here. It is not valid C++. Look carefully! You should be using the preprocessor `#if`.

Well, use the preprocessor #if